src/main/java/com/kainos/ea/model/JobRole.java

```diff
@@ -0,0 +1,47 @@
+package com.kainos.ea.model;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class JobRole
+{
+    private int roleID;
+
+    private String roleTitle;
+
+    private double salary;
+
+    @JsonCreator
+    public JobRole(@JsonProperty("salary") double salary,
+                   @JsonProperty("roleID") int roleID,
+                   @JsonProperty("role_title") String roleTitle )
```

Comment:
Looks like you've got an extra space there
```suggestion
                   @JsonProperty("role_title") String roleTitle)
```